automation/test-windows.sh

```diff
@@ -123,7 +123,7 @@ for size in ${sizes[@]}; do
     set +e
     current_time=0
     # run ipconfig /all command on windows vm
-    while [[ $(_oc exec -it winrmcli -- ./usr/bin/winrm-cli -hostname $ipAddressVMI -port 5985 -username "Administrator" -password "Heslo123" "ipconfig /all") != *"$ipAddressVMI"* ]] ; do 
+    while [[ $($(_oc exec -it winrmcli -- ./usr/bin/winrm-cli -hostname $ipAddressVMI -port 5985 -username "Administrator" -password "Heslo123" "ipconfig /all") | grep "IPv4 Address") != *"$ipAddressVMI"* ]] ; do 
```

Comment:
I'm not sure you need the double shell exec (nested $()) here. Couldn't you just use a pipe?

Reply:
The nesting is actually wrong, it will first generate new command "output of winrm | grep" and execute it

Reply:
fixed